vm/ubpf_run.c

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 Eideticom
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#define _GNU_SOURCE
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+//#include <stdbool.h>
+//#include <stdarg.h>
+//#include <inttypes.h>
+//#include <sys/mman.h>
+#include "ubpf_int.h"
+
+uint64_t ubpf_run(struct ubpf_vm *vm, void *mem, size_t mem_len, bool jit)
```

Comment:
@lsgunth I would like this added to the library so it can be used in other places (like eid-hermes perhaps). But I will move it into the ubpf_vm.c file.